server/application/application.go

```diff
@@ -574,6 +574,7 @@ func (s *Server) GetManifests(ctx context.Context, q *application.ApplicationMan
 				manifestInfo.Manifests[i].CompiledManifest = string(data)
 			}
 		}
+		manifests.SourcesManifestsStartingIdx = append(manifests.SourcesManifestsStartingIdx, int32(len(manifests.Manifests)))
```

Comment:
why you need this field? should we push it to OSS ?

Reply:
I need it to be able to report correct repo url, etc. for multi-sourced apps

Reply:
I think this info is useful for OSS also